test/unit/acl/access-groups.test.ts

```diff
@@ -0,0 +1,12 @@
+jest.mock('rdflib')
+import * as RdfLib from 'rdflib'
+jest.mock('solid-auth-client')
+import * as SolidAuthClient from 'solid-auth-client'
+
+import * as AccessGroups from '../../../src/acl/access-groups'
+
+describe('AccessGroups', () => {
+  it('exists', () => {
+    expect(AccessGroups).toBeTruthy()
```

Comment:
There is no way this could ever not be truthy - if you call `import * as AccessGroups`, then even if it does not export anything you would get `{}`. (Likewise in other tests.)

Reply:
Right, good point, I'll fix those tomorrow.

Reply:
Done